config/recommended.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     'event-sub-process-typed-start-event': 'error',
     'fake-join': 'warn',
     'label-required': 'error',
+    'no-missing-di': 'error',
```

Comment:
I guess it makes sense to put this into the `error` category. It is dangerous and deceiving for users, after all.

Reply:
:+1: